.github/workflows/action-conda-build.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: conda-incubator/setup-miniconda@v2
         with:
           python-version: ${{ matrix.python-version }}
-          miniconda-version: "latest"
+          miniconda-version: "py39_4.9.2"
```

Comment:
Why are you pinning to such a specific Python version? This will require manual updating much more often

Reply:
good catch! I will replace it with `latest` right now since we've seen it work well on Circle with a generic miniconda3-latest, cheers

Reply:
replaced it with the `latest` generic and the [build](https://github.com/ESMValGroup/ESMValTool/pull/2096/checks?check_run_id=2226355929) on GA runs fine, in the expected 40 odd minutes :+1: